content/en/community/code/_index.md

```diff
@@ -400,17 +400,32 @@ after the commit message has been created by the user.
 
 ### The commit message
 
-Jenkins X uses [conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/) as it's commit message format. These are particularly important as semantic releases are in use, and they use the commit messages to determine the type of changes in the codebase. Following formalized conventions for commit messages the semantic release automatically determines the next [semantic version](https://semver.org) number and generates a changelog based on the conventional commit.
+Jenkins X uses [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) message format.
 
-Semantic releases originate in the [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines), and the rules described there are the ones used by Jenkins X.
```

Comment:
I think we should keep this link, so that end users can see which other projects use it. 
They can go to pull requests in angular repo to get more insights imo.

Reply:
It is better to point to scripts that do the automation in `jx` itself. Following Angular docs is confusing for a person who just want to get own PR merged, because `jx` doesn't use anything except version bump.

Reply:
Not sure how it is confusing, as we only link to their commit message guideline :thinking: 
Can we put it at the bottom after we have described our guidelines. 
For example:
`More examples from another project can be seen here: <insert-angular-link>`

Reply:
On second thought, we do include the conventional commit link, so it should be ok. We can always add it back, if there are confusions, but I think this PR is fine for now ...

Reply:
Thanks. I was not sure I'd be getting time to get back to polish this PR,